app/graphql/queries/consultation/list.rb

```diff
@@ -0,0 +1,17 @@
+module Queries
+	module Consultation
+		class List < Queries::BaseQuery
+	    description "Get a list of consultations"
+	    argument :status_filter,		String, required: false, default_value: nil
```

Comment:
IS this going to be single select or multi select? same for ministry_filter .

Reply:
Single

Reply:
okay cool bro.